test.sh

```diff
@@ -37,6 +37,8 @@ k_wait_all_running() { bash ci/wait-pods-ok; }
     kubectl describe VirtualMachineInstanceMigration testvm-migration
   fi
 
+  kubectl delete mutatingwebhookconfigurations virt-api-mutator
```

Comment:
You need to do this: kubectl delete apiservices v1alpha3.subresources.kubevirt.io. Deleting the webhooks is of course not wrong but will not help. The webhooks don't block anything.

Reply:
Also you have to first delete the kubevirt CR. Otherwise the resources might get recreated anytime. In practice probably only if virt-api gets restarted ...